how-we-work/index.html

```diff
@@ -0,0 +1,116 @@
+---
+layout: blank
+title: How we work
+description: We use the SCRUM methodology and we’ll be happy to introduce its principles to you.
+keywords: methodology, scrum, agile, flexible contract
```

Comment:
[request] I'm missing the redirect as was discussed. Otherwise, SEO issues will surely appear.

https://github.com/jekyll/jekyll-redirect-from